phockup.py

```diff
@@ -149,50 +173,105 @@ def main():
 
 To get all date fields available for a file, do:
     exiftool -time:all -mimetype -j <file>
-        """,
+""",
     )
 
+    exclusive_group_debug_silent = parser.add_mutually_exclusive_group()
 
-    parser.add_argument(
-        "-q",
+    exclusive_group_debug_silent.add_argument(
+        "--debug",
+        action="store_true",
+        default=False,
+        help="""\
+Enable debugging.
+""",
+    )
+
+    exclusive_group_debug_silent.add_argument(
         "--quiet",
         action="store_true",
-        help="""Run without output
-        """,
+        default=False,
+        help="""\
+Run without output.
+""",
+    )
+
+    parser.add_argument(
+        "--log-file",
```

Comment:
Change to "--log", otherwise options.log in line 240 will throw exception.

Reply:
@roykrikke can you please take a look at this and if there is missing test not catching this issue to add one. Of course if you feel like doing it.

Reply:
@ivandokov and @tianleiwu I will have a look add the issue, no problem. I will also check if there is a missing test case. But to be honest, at the highest level we are missing test cases anyway. Idea: let's work together with a few people to realize high-level test cases. Who wants to help with this?

Reply:
I did change the `--log-file` to `--log` as I had to fix something else and I couldn't run phockup without changing this. It didn't broke any tests so I guess I haven't break anything but it's work looking at.

Reply:
@ivandokov and @tianleiwu I cannot reproduce the reported error. 

In the current master branch all is fine.
[code]
    if options.log:
        logfile = os.path.expanduser(options.log)
        fh = logging.FileHandler(logfile)
        fh.setFormatter(formatter)
        logger.addHandler(fh)
[/code]

Reply:
>  did change the `--log-file` to `--log` as I had to fix something else and I couldn't run phockup without changing this. It didn't broke any tests so I guess I haven't break anything but it's work looking at.

Ha oke, now i get it again. Strange that your comment was not visible yesterday when I wrote the above. Oh well... issue solved lets continue...